src/ol_infrastructure/substructure/keycloak/__main__.py

```diff
@@ -356,6 +356,20 @@
                 views=["admin", "user"], edits=["admin", "user"]
             ),
         ),
+        keycloak.RealmUserProfileAttributeArgs(
+            name="emailOptIn",
+            display_name="${emailOptIn}",
+            validators=[
+                keycloak.RealmUserProfileAttributeValidatorArgs(
+                    name="integer-bool",
+                    config={"integer": {"min": 0, "max": 1}},
```

Comment:
This is throwing the following error:
```
error: keycloak:index/realmUserProfile:RealmUserProfile resource 'olapps-user-profile' has a problem: '' expected type 'string', got unconvertible type 'map[string]interface {}', value: 'map[max:1 min:0]'. Examine values at 'olapps-user-profile.attributes[5].validators'.
```

Reply:
Hopefully I fixed that - keycloak exports min/max as strings (although for some reason it's not consistent because `max-length` is an integer), if that still doesn't work (the error makes me think it might not), I'll just remove the validation entirely since it's not directly user input.